Examples/Tests/radiation_reaction/test_const_B_analytical/inputs

```diff
@@ -0,0 +1,66 @@
+#Automatically generated inputfile
+#Run check.py without arguments to regenerate
+#
```

Comment:
With particles at the exact center of the box, I am not sure how they will be assigned to OpenMP threads. Could you add `warpx.serialize_ics = 1` to this file to enforce reproducibility?

Reply:
Sure